modules/matterhorn-asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/AssetManagerWithSecurity.java

```diff
@@ -73,12 +77,52 @@ public AssetManagerWithSecurity(
     this.secSvc = secSvc;
   }
 
-  @Override public Snapshot takeSnapshot(String owner, MediaPackage mp) {
-    final AccessControlList acl = authSvc.getActiveAcl(mp).getA();
+  private boolean isAuthorizedByAcl(AccessControlList acl, String action) {
     final User user = secSvc.getUser();
     final Organization org = secSvc.getOrganization();
-    if (AccessControlUtil.isAuthorized(acl, user, org, WRITE_ACTION)) {
+    return AccessControlUtil.isAuthorized(acl, user, org, action);
```

Comment:
Line Indentation missing.

Reply:
HI @CGreweling 
I've removed the `if` statement on the line preceding `return AccessControlUtil....` (line 83), so no indentation is necessary. Unless indentation is necessary for a reason I'm failing to gather?

Reply:
Yes, sorry @slampunk  you are right.